wordpress-mobile/gutenberg-mobile
# Update bundles to support content structure on HTML request.

Fixes #506

To test:
 - Test the main WP-iOS app using this PR:https://github.com/wordpress-mobile/WordPress-iOS/pull/14305
 - Test the main WP-Android app using this PR: https://github.com/wordpress-mobile/WordPress-Android/pull/12186
 - Check that the GB editor launches and you can get content info without needing to have the Metro tool running.

PR submission checklist:
